feature/stats/detail/src/main/java/com/twix/stats/detail/StatsDetailRoute.kt

```
@@ -0,0 +1,7 @@
+package com.yapp.stats.detail
+
+import androidx.compose.runtime.Composable
+
+@Composable
+fun StatsDetailRoute(onBack: () -> Unit) {
+}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)